docs/walkthroughs/face-to-face.md

```diff
@@ -0,0 +1,20 @@
+## Required docker images
+In order to reduce bandwidth consumption on the day, please pre-pull the required docker images before attending the Face 2 Face, using the following command:
+```
+for i in quay.io/3scale/apicast:master feedhenry/cordova-app-apb:0.0.6 feedhenry/ios-app-apb:0.0.6 feedhenry/android-app-apb:0.0.6 feedhenry/mcp-standalone:0.0.6 feedhenry/keycloak-apb:0.0.6 feedhenry/3scale-apb:0.0.6 feedhenry/aerogear-digger-apb:0.0.6 ansibleplaybookbundle/origin-ansible-service-broker:sprint139.1 openshift/origin-haproxy-router:v3.7.0-rc.0 openshift/origin-deployer:v3.7.0-rc.0 openshift/origin:v3.7.0-rc.0 openshift/origin-docker-registry:v3.7.0-rc.0 openshift/origin-pod:v3.7.0-rc.0 openshift/origin-service-catalog:v3.7.0-rc.0 quay.io/coreos/etcd:latest jimmidyson/keycloak-openshift:2.5.4.Final rhmap/redis:2.18.22; do docker pull $i; done
```

Comment:
We'll also need `aerogear/digger-android-slave-image:AGDIGGER-177` and `aerogear/digger-android-sdk-image:FH-v3.19` and `feedhenry/aerogear-digger-apb:0.0.6`

Reply:
Cheers @aidenkeating `feedhenry/aerogear-digger-apb:0.0.6` is already there, I've added the other 2 now.